addons/cubefs/v3.2/runtime-profile.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: data.fluid.io/v1alpha1
+kind: ThinRuntimeProfile
+metadata:
+  name: cubefs
```

Comment:
I suggest naming the profile name as cubefs3.2

Reply:
ok,  v2.4 runtime-profile also modify